### ISSUE TITLE
Live demo on website

### ISSUE THREAD

**Issue by bojidar-bg** [Hardmath123/nearley#68]
It will be very nice if it is possible to test Nearly online just like pegjs ([link](http://pegjs.org/online)).

**Comment by Hardmath123** [comment]
This is actually implemented, but commented out because it's slightly buggy. If you're interested, I can put some time into setting it up again.

**Comment by bojidar-bg** [comment]
Oh... now I notice this issue is a dupe of #37 :smile:

**Comment by bojidar-bg** [comment]
@Hardmath123 BTW, how should I compile a grammar in the browser (or without nearleyc)?

**Comment by Hardmath123** [comment]
Do it the way it's done in the commented-out section of the webpage. Here's some sample code: https://github.com/Hardmath123/nearley/blob/gh-pages/www/main.js#L6. You want to include all of these JS files: https://github.com/Hardmath123/nearley/blob/gh-pages/index.html#L9. Let me know if you use these features for something exciting!

**Comment by bojidar-bg** [comment]
Thanks for the links! I'll try to the best I can out of them :wink:

**Comment by Hardmath123** [comment]
Moving discussion to #106.

**Comment by Hardmath123** [comment]
See http://omrelli.ug/nearley-playground/

**Comment by bojidar-bg** [comment]
Yeah, I actually noticed this yesterday while browsing around the repo searching for a link... :smiley:
